config/env/default.js

```diff
@@ -370,6 +370,11 @@ module.exports = {
 	 */
 	maxTimeMS: 30000,
 
+	/*
+	 * The maximum time in milliseconds allowed for a count operation on the cursor by a mongo search/pagination query
+	 */
+	maxCountTimeMS: 5000,
```

Comment:
Based on my testing, if you are hitting a 5s count, you're at about 5M documents-- that's without projection or filtering.